.github/workflows/test-pr.yml

```diff
@@ -85,27 +85,27 @@ jobs:
         include:
           - name: 'Rules (booster)'
             test-suite: 'test-prove-rules'
-            test-args:
+            test-args: '--force-sequential'
```

Comment:
Shouldn't these arguments be baked into the Python tests so that locally we execute the same tests?

Reply:
Yes, they should.